app/controllers/pet_applications_controller.rb

```diff
@@ -0,0 +1,32 @@
+class PetApplicationsController < ApplicationController
+  def new
+  end
+
+  def create
+    @application = PetApplication.create(name: params[:name], street: params[:street], city: params[:city], state: params[:state], zip: params[:zip], description: params[:description])
+    if @application.save
+      @application.status_change("In Progress")
```

Comment:
For the status change, I think it is much easier to set a default value for this column in your migration, then doing it here in the controller.